Lesson20/selcan/oop.js

```diff
@@ -0,0 +1,106 @@
+// 1. Create a Person class with name, age, and gender properties, and a greet() method that logs a greeting message to the console. Then create a Student class that extends the Person class and adds a major property. Override the greet() method in the Student class to log a different greeting message to the console. Create 2 variables, one of type Person and one of type Student and log them to console.
+
+class Person {
+    constructor(name,age,gender){
+        this._name = name;
+        this._age = age;
+        this._gender = gender;
+    }
+    greet(){
+        return `Hi!My name is ${this._name}.I'm ${this._age} years old.I'm a ${this._gender}.`
+    }
+}
+class Student extends Person {
+    constructor(name,age,gender){
```

Comment:
Class student should also have a 'major' property, so you need to add it to the constructor method:
```
    constructor(name,age,gender, major){
      super(name,age,gender);
      this._major = major;
}
```